docusaurus/docs/cloud/projects/overview.md

```diff
@@ -43,11 +41,9 @@ From this tab you can:
 
 * **Trigger deploy**: Click the **Trigger deploy** button to trigger a new deployment
 * **Visit App**: Click the **Visit App** button to access the application
-* Quickly view project [**Usage**](./setting#usage)
+* Quickly view project [**Usage**](../getting-started/usage-billing)
```

Comment:
Please avoid relative links, as these are likely to break.
```suggestion
* Quickly view project [**Usage**](/cloud/getting-started/usage-billing)
```
I updated the link for you :)